spec/src/main/asciidoc/architecture.adoc

```diff
@@ -152,7 +154,7 @@ Global tags and tags registered with the metric are included in the output retur
 
 Global tags MUST NOT be added to the `MetricID` objects. Global tags must be included in list of tags when metrics are exported.
 
-NOTE: In application servers with multiple applications deployed, there is one reserved tag name: `_app`, which serves for
+NOTE: In application servers with multiple applications deployed, there is one reserved tag name: `mp_app`, which serves for
```

Comment:
This does not line up with the addition of `mp_scope` so there is no longer just _one_ reserved tag name. Maybe change this to something like:

NOTE: In application servers with multiple applications deployed, values of the reserved tag `mp_app` distinguish metrics from...